src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java

```diff
@@ -53,6 +53,7 @@ public Configuration() {
     public boolean removeIslandBlocksOnDelete = false;
     public boolean clearInventoryOnRegen = false;
     public boolean clearEnderChestOnRegen = false;
+    public boolean payPerRegen = false;
```

Comment:
Can we not remove this config option, and instead disable it if the schematic cost <= 0?

Reply:
aye aye, captain